### DISCUSSION TITLE
Android app - System notifications

### DISCUSSION THREAD

**Discussion by ml263** [meshtastic/Discussions#147] [Android]
Greetings community, 

I have been gifted a meshtastic node by a good friend so I'm a newbie right now. Having installed the android app. My Android notifications are a bit wild for now when I'm connected to my node using the app.  There are some notifications which would be useful to see, such as the uptime and stats one, or when I get a message. But the constant list that happened yesterday evening of pings for new nodes or stations was just crazy.  I looked and searched around for a while as to where I can edit this setting within the meshtastic app, but can't find it. I've seen posts on external notifications - but this seems to be for hardware add-ons or gpio outputs. 
The only other way I can see for now is to block the app from permission to use android notifications entirely within the system settings on the phone. 

The other problem I've seen is that when I get a long list of new stations/node I have to individually clear each notification, they can't be cleared as a group in one swipe. (not the end of the world, but also frustrating).

Anyway, I am enjoying having a play and learning. Thanks team for the support

ml263

**Comment by madeofstown** [comment]
If you long press on the Meshtastic service notification:
![Screenshot_20241226-080934](https://github.com/user-attachments/assets/6a7c68d6-369d-46b3-9967-9603a8f99682)
This menu will come up:
![Screenshot_20241226-080945](https://github.com/user-attachments/assets/073588f1-90f1-472c-a0bf-df963c206e58)
Then if you click the gear icon on the top corner you get this screen:
![Screenshot_20241226-080953](https://github.com/user-attachments/assets/96178cc1-365b-4c72-9b6a-9b583fd5d43d)

**Reply by ml263** [reply]
Thanks for the reply. 
I think that's the answer I was looking for. I was familiar with the first step you mentioned, but not the second step you allowing to refine the notifications allowed. 

Thanks very much 👍

ml263

**Comment by easytarget** [comment]
> familiar with the first step you mentioned, but not the second step you allowing to refine the notifications allowed.

Similar here; and I suspect plenty of others are wondering how to quieten the notifications. This is a prime candidate for a 'getting started with android app' document, or in the main app menu itself.
